test/test_translator.py

```diff
@@ -138,8 +136,73 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_empty_string() :
```

Comment:
Since we're not testing and empty string value here (''), we should name this test something like `test_parse_polyline_null_parameter` or simliar.

Reply:
I changed the name to test_parse_polyline_null_parameter  and pushed it to dev